pmm-app/src/pmm-add-instance/components/Discovery/Discovery.service.ts

```diff
@@ -0,0 +1,15 @@
+import { apiRequestManagement } from 'shared/components/helpers/api';
+
+class DiscoveryService {
+  static async discoveryRDS({ aws_access_key, aws_secret_key }) {
+    // return {
```

Comment:
Remove comment. A return type can be added instead of any

Reply:
done